[Readme instructions] Your OpenSSL implementation does not include the RIPEMD160 algorithm

After running this step, the issue was encountered:
```
kmnt23@kmnt23-desktop:~/Workspace/xrpl$ python3 tools/account_info.py
Traceback (most recent call last):
  File "/home/kmnt23/Workspace/xrpl/tools/account_info.py", line 5, in <module>
    from xrpl.clients import JsonRpcClient
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/__init__.py", line 2, in <module>
    from xrpl import account, clients, core, ledger, models, transaction, utils, wallet
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/account/__init__.py", line 2, in <module>
    from xrpl.account.main import (
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/account/main.py", line 6, in <module>
    from xrpl.asyncio.account import main
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/asyncio/__init__.py", line 2, in <module>
    from xrpl.asyncio import account, clients, ledger, transaction, wallet
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/asyncio/account/__init__.py", line 2, in <module>
    from xrpl.asyncio.account.main import (
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/asyncio/account/main.py", line 5, in <module>
    from xrpl.asyncio.clients import Client, XRPLRequestFailureException
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/asyncio/clients/__init__.py", line 2, in <module>
    from xrpl.asyncio.clients.async_json_rpc_client import AsyncJsonRpcClient
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/asyncio/clients/async_json_rpc_client.py", line 2, in <module>
    from xrpl.asyncio.clients.async_client import AsyncClient
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/asyncio/clients/async_client.py", line 4, in <module>
    from xrpl.asyncio.clients.client import Client
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/asyncio/clients/client.py", line 6, in <module>
    from xrpl.models.requests.request import Request
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/models/__init__.py", line 2, in <module>
    from xrpl.models import amounts, currencies, requests, transactions
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/models/requests/__init__.py", line 34, in <module>
    from xrpl.models.requests.sign import Sign
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/models/requests/sign.py", line 26, in <module>
    from xrpl.models.transactions.transaction import Transaction
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/models/transactions/__init__.py", line 6, in <module>
    from xrpl.models.transactions.account_delete import AccountDelete
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/models/transactions/account_delete.py", line 7, in <module>
    from xrpl.models.transactions.transaction import Transaction
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/models/transactions/transaction.py", line 10, in <module>
    from xrpl.core.binarycodec import encode
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/core/__init__.py", line 2, in <module>
    from xrpl.core import addresscodec, binarycodec, keypairs
  File "/home/kmnt23/.local/lib/python3.10/site-packages/xrpl/core/keypairs/__init__.py", line 16, in <module>
    assert (
AssertionError: Your OpenSSL implementation does not include the RIPEMD160 algorithm,
    which is required by XRPL
```
I had to apply this [solution](https://stackoverflow.com/questions/69922525/python-3-9-8-hashlib-and-ripemd160/72509045#72509045) to make this step work on Mint 2.1

Maybe Readme.md could be updated.

--- a/README.md
+++ b/README.md
@@ -38,6 +38,26 @@ Follow the steps below to save an initial state that can be loaded later in cert
 1. Make sure you have python3 installed. You should be able to run `python3 --version`.
 2. Install `xrpl` python lib: `pip3 install xrpl-py`.
 
+##### Important note!
+The `xrlp` library depends on legacy openSSL functions that are disabled by default. In the case of error, make sure to explicitly enable legacy functions in the config file like so:
+   ```
+   openssl_conf = openssl_init
+   
+   [openssl_init]
+   providers = provider_sect
+   
+   [provider_sect]
+   default = default_sect
+   legacy = legacy_sect
+   
+   [default_sect]
+   activate = 1
+   
+   [legacy_sect]
+   activate = 1
+   ```
+You can find out the path to the config file with the following command: `openssl version -d`.
+
 ##### Mac users
 Make sure these two `127.0.0.x` (where `x != 1`) addresses are enabled:
 ```

